extension/src/popup/ducks/transactionSubmission.ts

```diff
@@ -95,12 +94,26 @@ export const submitFreighterTransaction = createAsyncThunk<
   "submitFreighterTransaction",
   async ({ signedXDR, networkDetails }, thunkApi) => {
     try {
-      const txRes = await internalSubmitFreighterTransaction({
-        signedXDR,
-        networkDetails,
-      });
+      const options = {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          signed_xdr: signedXDR,
+          network_url: networkDetails.networkUrl,
+          network_passphrase: networkDetails.networkPassphrase,
+        }),
+      };
+      const res = await fetch(`${INDEXER_URL}/submit-tx`, options);
+      const response = await res.json();
 
-      return txRes;
+      if (res.status !== 200) {
```

Comment:
NIT: this should probably check `res.ok` which will work for all 2xx responses

Reply:
oh yeah good call, added in 141c56475007f1899203f5255ccadde1a6ab9485